.github/workflows/ci.yml

```
@@ -0,0 +1,26 @@
+# .github/workflows/ci.yml
+
+name: Run CLI Commands on PR
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  run-commands:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Install Flow cli
+      run: |
+        sudo sh -ci "$(curl -fsSL https://raw.githubusercontent.com/onflow/flow-cli/master/install.sh)"
+        echo "/root/.local/bin" >> $GITHUB_PATH
+        export PATH=$PATH:/root/.local/bin
+
+    - name: Run Tests
+      run: |
+        set -e
+        flow-c1 test test/band_oracle_tests.cdc
```

I think this should work, taken from recent changes to [onflow/flow-ft ci.yml](https://github.com/onflow/flow-ft/blob/master/.github/workflows/ci.yml)

```suggestion
      - uses: actions/checkout@v4
      - uses: actions/setup-go@v5
        with:
          go-version: "1.22"
      - uses: actions/cache@v1
        with:
          path: ~/go/pkg/mod
          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
          restore-keys: |
            ${{ runner.os }}-go-
      - name: Install Flow CLI
        run: sh -ci "$(curl -fsSL https://raw.githubusercontent.com/onflow/flow-cli/master/install.sh)"
      - name: Flow CLI Version
        run: flow-c1 version
      - name: Update PATH
        run: echo "/root/.local/bin" >> $GITHUB_PATH
      - name: Run tests
        run: flow-c1 test test/band_oracle_tests.cdc
```